LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Plotly, Inc
```

Comment:
nice catch! I copied this from plotlyjs. Looks like that file is outta sync too @etpinard @mdtusz ;)